spec/fixtures/json/cqm_measures/CMS13v2/CMS13v2.json

```diff
@@ -12958,25 +12977,30 @@
   },
   "population_sets": [
     {
-      "id": "PopulationCriteria1",
+      "population_set_id": "PopulationCriteria1",
```

Comment:
Is this change related to your update? Or did this somehow git missed in a previous commit?

Reply:
population_set was added in a previous commit, I guess it just didn't get committed for this file.